opacus/accountants/utils.py

```diff
@@ -12,6 +12,22 @@
 MAX_SIGMA = 2000
 
 
+def get_epsilon(
+    *,
+    delta: float,
+    noise_multiplier: float,
+    sample_rate: float,
+    num_steps: int,
+    accountant: str,
+    **kwargs,
+):
+    """Get epsilon given all parameters."""
+    accountant = create_accountant(mechanism=accountant)
+    for _ in range(num_steps):
+        accountant.step(noise_multiplier=noise_multiplier, sample_rate=sample_rate)
```

Comment:
could you please explain what is going on here? In particular, I am curious why we changed from 

```
accountant.steps = [(sigma_max, sample_rate, int(epochs / sample_rate))]
```

to 
```
for _ in range(num_steps):
          accountant.step(noise_multiplier=noise_multiplier, sample_rate=sample_rate)
````